src/diagnostics/MOM_diagnostics.F90

```diff
@@ -83,6 +83,7 @@ module MOM_diagnostics
   integer :: id_PE_to_KE       = -1, id_KE_BT          = -1
   integer :: id_KE_Coradv      = -1, id_KE_adv         = -1
   integer :: id_KE_visc        = -1, id_KE_stress      = -1
+  integer :: id_KE_visc_gl90 = -1
```

Comment:
Since this file needs to be edited anyway, maybe edit this line to match the style of adjacent lines with respect to indentation.

(Normally I would not raise this, but since you need to modify the expression below, might as well)